ClosedXML/Excel/IXLWorkbook.cs

```diff
@@ -223,6 +223,14 @@ public interface IXLWorkbook : IDisposable
 
         XLWorkbook SetUse1904DateSystem(Boolean value);
 
+        /// <summary>
+        /// Gets the Excel table of the given name
+        /// </summary>
+        /// <param name="tableName">Name of the table to return.</param>
+        /// <param name="comparisonType">One of the enumeration values that specifies how the strings will be compared.</param>
+        /// <returns>The table with given name or </returns>
```

Comment:
The table with given name or _what_? The implementation below throws the exception if not found.

Reply:
ok done